test/tools/evmoneRunner.h

```diff
@@ -0,0 +1,48 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include <deps/evmone/include/evmone/evmone.h>
+#include <string>
+
+namespace external
+{
+namespace evmone
+{
+class EvmOneVM : public evmc_context
+{
+public:
+	EvmOneVM() : evmc_context{&interface}, vm{evmc_create_evmone()} {}
+	// Wrapper for evmone::execute.
+//	void execute(int64_t _gas, std::string _hexCode, std::string _input);
```

Comment:
Why are there so many?

Reply:
I initially chose a single wrapper from here but ran into runtime issues that I wasn't able to debug. Finally, I ended up adopting the same wrapper waterfall that evmone's unit tests use and after that the problem I was running into magically disappeared :-)

https://github.com/chfast/evmone/blob/master/test/unittests/execution_fixture.hpp